huang_van_genuchten_data/README.md

```diff
@@ -0,0 +1,13 @@
+# Van Genuchten data from Huang, 2011
+
+This artifact contains soil van Genuchten parameters as a function of depth for
+soil from site SV62 in Fort McMurray, Alberta, Canada.
+
+The data was originally collected by Huang, Mingbin, et al. [1], and is
+available in Table 1B.
```

Comment:
To me, "it is available in Table 1B" implies the user should get the data from there, rather than by running `create_artifact.jl`. Maybe we could clarify that we're providing access to it via that script

Reply:
Yeah, just to be clear, this came from their paper (so all I did was transcribe it)

Reply:
Added a note about this